core/templates/dev/head/domain/exploration/ReadOnlyExplorationBackendApiServiceSpec.js

```diff
@@ -107,9 +75,9 @@ describe('Read only exploration backend API service', function() {
     var successHandler = jasmine.createSpy('success');
     var failHandler = jasmine.createSpy('fail');
 
-    $httpBackend.expect('GET', '/createhandler/data/0').respond(
+    $httpBackend.expect('GET', '/explorehandler/init/0').respond(
       sampleDataResults);
-    ReadOnlyExplorationBackendApiService.fetchExploration('0').then(
+    ReadOnlyExplorationBackendApiService.fetchExploration('0', null).then(
```

Comment:
Do we have any tests verifying the different behavior when a version is provided? Caching doesn't happen, etc.